### DISCUSSION TITLE
Questions about background

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#3781] [Q&A]
Hello Artf.

Two questions that i didn't find if it's a bug or by design.

1 - When select the body, we only have background image to set. So, we can't set background color as background, also, gradient plugin didn't work to body.
Print from online current demo:
![image](https://user-images.githubusercontent.com/2287371/133616272-79275c02-c224-480e-a2df-427f39f7ef94.png)

1.1 - But, when using the mjml plugin, i can set it as image and color.
Print from my current implementation:
![image](https://user-images.githubusercontent.com/2287371/133616586-7decd1e6-7c07-4dd8-b541-32b152e5275d.png)
And a print from current online demo:
![image](https://user-images.githubusercontent.com/2287371/133616739-7d908b1d-1730-4fa7-bd23-79b7f86c9fc2.png)

2 - I was looking into the plugin grapesjs-style-bg, and I realized that the spectrum implementation haven't the input to set the hex color as the sample provided in the grapick plugin. 
2.1 - I can't find it in the code to know if it's a bug or by design again.
2.2 - These plugin didn't cancel the color changed when clicked on "x" button, check the video.

https://user-images.githubusercontent.com/2287371/133618689-fa9a1e10-1280-4d3f-b904-8488a6cbe446.mp4

**Comment by LongEcho** [comment]
I've been wondering about that too. 

But found an **old version** where colors are supported:
https://github.com/artf/grapesjs/tree/3ff352cc5cc7b3c526dd272ece3811fbf4e519f8
(Download as zip, open demo.html)
As far as I know there are **no** real **drawbacks** compared to the new version.

Or you can just use the **Newsletter Demo**:
https://grapesjs.com/demo-newsletter-editor.html
It has color Background support in it's **latest version**.

I hope this helps
